cmd/config.toml

```
@@ -0,0 +1,44 @@
+[global_tags]
+  # dc = "us-east-1" # will tag all metrics with dc=us-east-1
+  # rack = "1a"
+  ## Environment variables can be used as tags, and throughout the config file
+  # user = "$USER"
+customer_id = "X"
+service = "MP"
```

Indentation still wrong, no?

```suggestion
  customer_id = "X"
  service = "MP"
```